src/tests/editor.test.js

```diff
@@ -1698,7 +1698,7 @@ describe('Editor', () => {
                 await testEditor(BasicEditor, {
                     contentBefore: '<h1>[<b>abcd</b></h1><p>ef]gh</p>',
                     stepFunction: deleteBackward,
-                    contentAfter: '<h1>[]gh</h1>',
+                    contentAfter: '<h1><b>[]</b>gh</h1>',
```

Comment:
Now that I think about it, this might actually be more correct that it used to be. Visually for the user, the starting content would be no different if the start was *inside* the `b`, so I think this is perfectly acceptable, not even a true tradeoff.

Reply:
_Now_ you're starting to understand. :t-rex: